docs/releases/1.20-NOTES.md

```diff
@@ -20,6 +22,8 @@
 
 * If you are running `kops toolbox template` in an airgapped environment, you have to set `--channel` to point to a local channel file.
 
+* If your workload targets control plane nodes, you need to change them to select the `node-role.kubernetes.io/control-plane=""` label. You should also add the `node-role.kubernetes.io/control-plane:NoSchedule` toleration to these workloads. This taint will not be added to control plane nodes before kOps 1.22.
```

Comment:
Hi 👋 

I arrived here from 1.23 release notes: https://github.com/kubernetes/kops/blob/v1.23.0/docs/releases/1.23-NOTES.md?plain=1#L48

I'm running kOps v1.22.2 and can confirm there is no matching taint as suggested in this PR. So when I replace my `node-role.kubernetes.io/master:NoSchedule` toleration as suggested here, my pod won't schedule.

Is it possible someone forgot to add the new taint? Or, if was it added in between v1.22 and v1.23, can someone point me to the source code?

Many thanks!

Reply:
The notes say you need both taints to be forward proof